apps/web/ui/partners/partner-application-details.tsx

```
@@ -13,26 +14,63 @@ export function PartnerApplicationDetails({
+        case "long-text":
+          return {
+            title: field.label,
+            value: field.value,
+          };
+        case "select":
+          return {
+            title: field.label,
+            value: field.value,
+          };
+        case "multiple-choice":
+          let value;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)